doc/HttpLuaModule.wiki

```diff
@@ -6630,10 +6630,38 @@ This feature was first introduced in the <code>v0.10.7</code> release.
 
 '''context:''' ''rewrite_by_lua*, access_by_lua*, content_by_lua*, ngx.timer.*, ssl_certificate_by_lua*, ssl_session_fetch_by_lua*''
 
-This function is added for [http://w3.impa.br/~diego/software/luasocket/tcp.html LuaSocket] API compatibility and does nothing for now. Its functionality will be implemented in future.
+This function is added for [http://w3.impa.br/~diego/software/luasocket/tcp.html LuaSocket] API compatibility and does nothing for now. Its functionality is implemented <code>v0.10.18</code>.
 
 This feature was first introduced in the <code>v0.5.0rc1</code> release.
 
+<code>option</code> is a string with the option name, and value depends on the option being set:
+
+* <code>keepalive</code>
+: Setting this option to true enables sending of keep-alive messages on
+: connection-oriented sockets(e.g., tcpsock:setoption("keepalive", 1) and
+: tcpsock:setoption("keepalive", false)).
+
+* <code>reuseaddr</code>
+: Enable this option indicates that the rules used in validating addresses
```

Comment:
Grammar: "enabe" is a verb, which cannot be used as the subject of the sentence. Always check the grammar of your newly added docs.

Reply:
accept